open-webui/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: open-webui_web_1
+      APP_PORT: 8080
+      PROXY_AUTH_ADD: "false"
```

Comment:
I think it makes sense to disable the proxy authentication here because Open WebUI has its own built-in authentication that includes an initial set-up for an admin user.

I checked and it is possible to disable open-webui's in-built authentication by adding ` WEBUI_AUTH: "False"` as an env var to the open-webui container if we'd rather have umbrelOS proxy auth instead, but then we lose the ability in OPen WebUI to have multiple users with the admin user behind a separate password.